tornado/auth.py

```diff
@@ -848,8 +848,8 @@ class GoogleOAuth2Mixin(OAuth2Mixin):
 
     .. versionadded:: 3.2
     """
-    _OAUTH_AUTHORIZE_URL = "https://accounts.google.com/o/oauth2/auth"
-    _OAUTH_ACCESS_TOKEN_URL = "https://accounts.google.com/o/oauth2/token"
+    _OAUTH_AUTHORIZE_URL = "https://accounts.google.com/o/oauth2/v2/auth"
+    _OAUTH_ACCESS_TOKEN_URL = "https://www.googleapis.com/oauth2/v4/token"
```

Comment:
Is there any detail about what has changed between the different versions of this URL? It makes me nervous to (presumably) jump over several versions at once, although I guess the existence of the "discovery document" protocol implies that these URL versions shouldn't really make a difference for clients.